gooddata-sdk/gooddata_sdk/utils.py

```diff
@@ -424,3 +425,8 @@ def ref_extract(ref: dict[str, Any]) -> Union[str, ObjId]:
         return ref["localIdentifier"]
 
     raise ValueError("invalid ref. must be identifier or localIdentifier")
+
+
+def filter_for_attributes_labels(attributes: list[Attribute]) -> str:
+    label_ids = [attribute.label.id for attribute in attributes]
```

Comment:
Would make this a set, there can be duplicates in the execution, but no need to include them multiple times.

Reply:
Good point. I tried to solve it as well. Unfortunately, `set()` cannot be used because the order is not deterministic, but I found a way to remove duplicates.